src/main/java/org/kairosdb/core/http/rest/QueryProcessingChainRessources.java

```diff
@@ -0,0 +1,63 @@
+package org.kairosdb.core.http.rest;
+
+
+import com.google.common.collect.ImmutableList;
+import com.google.gson.Gson;
+import com.google.inject.Inject;
+import org.kairosdb.core.http.rest.json.JsonResponseBuilder;
+import org.kairosdb.core.processingstage.QueryProcessingChain;
+import org.kairosdb.core.processingstage.QueryProcessingStageFactory;
+import org.kairosdb.core.processingstage.metadata.QueryProcessingStageMetadata;
+import org.kairosdb.core.processingstage.metadata.QueryProcessorMetadata;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import static org.kairosdb.core.http.rest.MetricsResource.setHeaders;
+
+@Path("/api/v1/queryprocessing")
+public class QueryProcessingChainRessources
```

Comment:
Need to rename this class (QueryProcessingChainRes~s~ources), I've forgot to correct it.